psycop/common/sequence_models/tasks/tasks.py

```diff
@@ -167,6 +163,15 @@ def collate_fn(
         batch_with_labels = self.masking_fn(padded_sequence_ids)
         return batch_with_labels
 
+    def on_fit_start(self) -> None:
```

Comment:
This seems very dirty, but since the mlm_head is not initialised in __init__, it is not correctly initialised when loading from checkpoint otherwise. 

I considered using `setup`, but that re-initialised on every fit, predict etc., which I think means we'll lose the weights and biases of the mlm head.